lib/api.js

```diff
@@ -24,10 +23,21 @@ var certFile = path.resolve(__dirname, '../config/cert.pem');
 // Start express server
 var express = require('express');
 var app = express();
-var server = https.createServer({
-  key: fs.readFileSync(keyFile),
-  cert: fs.readFileSync(certFile),
-}, app);
+var server;
+
+// We default to https protocol, if certs are not found fallback to http
+var  protocol = 'https';
+try {
+  server = require('https').createServer({
+    key: fs.readFileSync(keyFile),
+    cert: fs.readFileSync(certFile),
+  }, app);
+} catch (e) {
+  server = require('http').createServer(app);
+  protocol = 'http';
+
+}
```

Comment:
I don't think try/catch is the best approach here b/c you could end up failing over to http due to some unrelated error. Instead I would test for existence of just the `certFile`. If it's there, do https, if not, do http. If the key is missing or cert is invalid or whatever, it will fail loudly, which is appropriate.